labels.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import requests
+import json
+from webcompat import app
+
+API_URI = 'https://api.github.com/repos/'
+ISSUES_URI = app.config['ISSUES_REPO_URI'].rpartition('/issues')[0]
+USER_LABELS_URI = API_URI + ISSUES_URI + "/labels"
+WEBCOMPAT_LABELS_URI = API_URI + 'webcompat/webcompat-tests/labels'
+AUTH_HEADERS = {'Authorization': 'token {0}'.format(app.config['OAUTH_TOKEN']),
+                'User-Agent': 'webcompat/webcompat-bot'}
+
+
+def get_issue_labels(labels_uri):
+    '''Get the issue labels from the given repo.'''
+    r = requests.get(labels_uri)
+    return r.json()
+
+
+def create_label(data):
+    '''Creates a label in the given repo'''
+    requests.post(USER_LABELS_URI, json.dumps(data), headers=AUTH_HEADERS)
+
+
+def delete_label(name):
+    '''Creates a label in the given repo'''
+    delete_labels_uri = USER_LABELS_URI + "/" + name
+    requests.delete(delete_labels_uri, headers=AUTH_HEADERS)
+
+
+'''delete all existing labels from the user issues repo'''
+user_labels = get_issue_labels(USER_LABELS_URI)
+for label in user_labels:
+    delete_label(label.get('name'))
+
+'''get all labels from the webcompat tests repo'''
+webcompat_labels = get_issue_labels(WEBCOMPAT_LABELS_URI)
+
+'''re-create labels in user issues repo'''
+for label in webcompat_labels:
```

Comment:
This should happen only if the  previous request has been successful. If not successful, we should warn there was an issue.